[Testerina] Test Report Test 'testWithoutCoverage' fails due to incorrect module coverage

**Description:**
The following test case runs after `testWithCoverage()`.

```
@Test (enabled = false)
    public void testWithoutCoverage() throws BallerinaTestException, IOException {
        runCommand(false);
        validateStatuses();
        Assert.assertEquals(resultObj.get("moduleCoverage").getAsJsonArray().size(), 0);
    }
```

The test case fails with the followng error 

```
testerina-intg-test-suite > testerina-intg-tests > org.ballerinalang.testerina.test.TestReportTest > testWithoutCoverage FAILED
    java.lang.AssertionError: expected [0] but found [2]
        at org.testng.Assert.fail(Assert.java:96)
        at org.testng.Assert.failNotEquals(Assert.java:776)
        at org.testng.Assert.assertEqualsImpl(Assert.java:137)
        at org.testng.Assert.assertEquals(Assert.java:118)
        at org.testng.Assert.assertEquals(Assert.java:652)
        at org.testng.Assert.assertEquals(Assert.java:662)
        at org.ballerinalang.testerina.test.TestReportTest.testWithoutCoverage(TestReportTest.java:66)
```
If the test case is run without module coverage then the expected value for `get("moduleCoverage")` should be zero.

**Steps to reproduce:**
Enable the `testWithoutCoverage()` test case and run the tests

**Affected Versions:**

**OS, DB, other environment details and versions:**

**Related Issues (optional):**
<!-- Any related issues such as sub tasks, issues reported in other repositories (e.g component repositories), similar problems, etc. -->

**Suggested Labels (optional):**
<!-- Optional comma separated list of suggested labels. Non committers can’t assign labels to issues, so this will help issue creators who are not a committer to suggest possible labels-->

**Suggested Assignees (optional):**
<!--Optional comma separated list of suggested team members who should attend the issue. Non committers can’t assign issues to assignees, so this will help issue creators who are not a committer to suggest possible assignees-->

--- a/tests/testerina-integration-test/src/test/java/org/ballerinalang/testerina/test/TestReportTest.java
+++ b/tests/testerina-integration-test/src/test/java/org/ballerinalang/testerina/test/TestReportTest.java
@@ -59,7 +59,7 @@ public class TestReportTest extends BaseTestCase {
         validateCoverage();
     }
 
-    @Test (enabled = false)
+    @Test ()
     public void testWithoutCoverage() throws BallerinaTestException, IOException {
         runCommand(false);
         validateStatuses();
@@ -71,7 +71,7 @@ public class TestReportTest extends BaseTestCase {
         if (coverage) {
             args = new String[]{"--code-coverage", "--all"};
         } else {
-            args = new String[]{ "--all"};
+            args = new String[]{"--test-report", "--all"};
         }
 
         balClient.runMain("test", args, null, new String[]{},

